src/themes/default/pages/Home.vue

```diff
@@ -1,8 +1,10 @@
 <template>
   <div id="home">
   <main-slider />
-  
-  <section class="new-collection container pt40 px15">
+
+  <promoted-offers></promoted-offers>
```

Comment:
replace with `<promoted-offers />` 
Additionally it should be PascalCase but remove close tag at least please.